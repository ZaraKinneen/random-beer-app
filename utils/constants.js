export const CORS = "https://cors-anywhere.herokuapp.com";
export const BASE_URL = "http://api.brewerydb.com/v2";
export const API_KEY = "a5c1b917e7ba62dcd79f434ed73bc72d";

export const IMAGE_PLACEHOLDER =
  "https://touch.daft.ie/static/images/fallbacks/no-image-size740x480.jpg";
