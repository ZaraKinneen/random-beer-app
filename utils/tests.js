import { render } from "@testing-library/react";
import { ThemeProvider } from "styled-components";

const theme = {
  colors: {
    primary: "#355C7D",
    secondary: "#C06C84",
  },
  fonts: {
    light: "#FFFFFF",
    dark: "#000000",
    highlight: "#FFCC00",
  },
};

const ThemeWrapper = ({ children }) => {
  return <ThemeProvider theme={theme}>{children}</ThemeProvider>;
};

const customRender = (ui, options) =>
  render(ui, { wrapper: ThemeWrapper, ...options });

// re-export everything
export * from "@testing-library/react";

// override render method
export { customRender as render };
