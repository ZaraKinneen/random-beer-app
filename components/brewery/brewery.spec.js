import { render } from "../../utils/tests";
import BreweryComponent from "./brewery";

describe("BreweryComponent", () => {
  it("should match snapshot", () => {
    const { container } = render(<BreweryComponent />);

    expect(container).toMatchSnapshot();
  });
});
