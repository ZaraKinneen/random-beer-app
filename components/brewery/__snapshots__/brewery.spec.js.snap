// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BreweryComponent should match snapshot 1`] = `
<div>
  <div
    class="brewery__Container-eggu3v-0 eSQSSF"
  >
    <span
      class="brewery__LinkContainer-eggu3v-8 Kdlpq"
      href="/beer/undefined"
    >
      <p
        class="brewery__Text-eggu3v-6 iABviU"
      >
        Go Back
      </p>
    </span>
    <div
      class="brewery__Main-eggu3v-1 NIkkH"
    >
      <div
        class="brewery__Grid-eggu3v-2 huEpEK"
      >
        <div
          class="brewery__Card-eggu3v-3 iYJFOE"
        >
          <h1
            class="brewery__Title-eggu3v-9 hvnLuT"
          />
          <span
            class="brewery__TextContainer-eggu3v-5 gQCchi"
          >
            <img
              alt="organic-img"
              class="brewery__Icon-eggu3v-7 hsbnXh"
              src="/images/grain.svg"
            />
            <p
              class="brewery__Text-eggu3v-6 iABviU"
            >
              Organic: 
            </p>
          </span>
          <span
            class="brewery__TextContainer-eggu3v-5 gQCchi"
          >
            <img
              alt="www-img"
              class="brewery__Icon-eggu3v-7 hsbnXh"
              src="/images/www.svg"
            />
            <p
              class="brewery__Text-eggu3v-6 iABviU"
            >
               
            </p>
          </span>
          <span
            class="brewery__TextContainer-eggu3v-5 gQCchi"
          >
            <img
              alt="establsihed-img"
              class="brewery__Icon-eggu3v-7 hsbnXh"
              src="/images/factory.svg"
            />
            <p
              class="brewery__Text-eggu3v-6 iABviU"
            >
              Established: 
               
            </p>
          </span>
        </div>
        <div
          class="brewery__Card-eggu3v-3 iYJFOE"
        >
          <img
            class="brewery__Image-eggu3v-4 dmcwMa"
          />
        </div>
        <div
          class="brewery__Card-eggu3v-3 iYJFOE"
        >
          <p
            class="brewery__Text-eggu3v-6 iABviU"
          >
             
          </p>
        </div>
      </div>
    </div>
  </div>
</div>
`;
